gradle.properties

```diff
@@ -5,11 +5,11 @@
 niagara_home=c:\\niagara\\niagara-4.14.0.162
 
 #The path to niagara_user_home for the version of Niagara you are building against
-niagara_user_home=
+niagara_user_home=C:\\Users\\nickvaladez\\Niagara4.14\\tridium
 
 #Uncomment and set to the path of your node install if you are building JavaScript
 #modules
-#nodeHome=
+nodeHome=C:\\Program Files\\nodejs
```

Comment:
Technically you only need to set this if you're building JavaScript, but there's nothing wrong with you updating it.